static/bidder-info/intertech.yaml

```diff
@@ -0,0 +1,14 @@
+endpoint: https://prebid.intertechsrvcs.com/prebid/{{page_id}}?imp-id={{imp_id}}&ssp-id=10500
+endpoint-compression: gzip
+meta-info:
+  maintainer-email: prebid@intertechsrvcs.com
+  site-media-types:
+    - banner
+    - native
+  vendor-id: 0
+usersync:
+  url: https://prebid.intertechsrvcs.com/mapuid/intertech/?ssp-id=10500&gdpr={{gdpr}}&gdpr_consent={{gdpr_consent}}&location=
+  redirect-url: /setuid?bidder=intertech&gdpr={{gdpr}}&gdpr_consent={{gdpr_consent}}&us_privacy={{us_privacy}}&uid={UID}
+  cookie-family-name: intertech
+  type: redirect
+  support-cors: false
```

Comment:
PBS-Go uses a slightly different format here than PBS-Java and the macros have slightly different names to follow the convention of Go templates. This should look like:

```
# This adapter is ported from PBS-Java to PBS-Go by Prebid. Please open a GitHub issue instead of 
# directly contacting the maintainer for technical issues.
endpoint: https://prebid.intertechsrvcs.com/prebid/{{.PageID}}?imp-id={{.ImpID}}&ssp-id=10500
endpointCompression: GZIP
maintainer:
  email: prebid@intertechsrvcs.com
capabilities:
  site:
    mediaTypes:
      - banner
      - native
userSync:
  redirect:
    url: https://prebid.intertechsrvcs.com/mapuid/intertech/?ssp-id=10500&gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&location={{.RedirectURL}}
    userMacro: "{UID}"
```

This is the first PBS-Go ported adapter. I recommend we add a comment at the start of some kind to indicate this is maintained by Prebid and not the adapter maintainer.

There is no existing ImpID macro. You can add one in this PR by expanding the EndpointTemplateParams struct.